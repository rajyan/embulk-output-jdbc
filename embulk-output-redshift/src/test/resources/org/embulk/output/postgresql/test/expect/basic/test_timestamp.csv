id:long,date_item:timestamp,time_item:timestamp,timestamp_item:timestamp
1,2019-1-2 00:00:00.000 +0900,2001-1-1 12:34:56.000 +0900,2019-12-31 23:59:59.000 +0900
2,,,
