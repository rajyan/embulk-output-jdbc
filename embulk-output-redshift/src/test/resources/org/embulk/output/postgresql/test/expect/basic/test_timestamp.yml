table: test_timestamp
mode: insert
