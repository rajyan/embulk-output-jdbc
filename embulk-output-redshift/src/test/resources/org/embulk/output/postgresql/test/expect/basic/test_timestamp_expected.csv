1,2019-01-01,03:34:56,2019-12-31 14:59:59
2,\N,\N,\N
